manager/controlapi/service_test.go

```diff
@@ -436,6 +436,18 @@ func TestRemoveService(t *testing.T) {
 	assert.NotNil(t, r)
 }
 
+func TestValidateEndpointSpec(t *testing.T) {
+	err := validateEndpointSpec(&api.EndpointSpec{
+		Mode: api.ResolutionModeDNSRoundRobin,
+		Ports: []*api.PortConfig{
+			{
+				Name: "http", TargetPort: 80,
+			},
+		},
+	})
+	assert.Error(t, err)
```

Comment:
I don't see that other tests validate error messages. I've seen this though

`assert.Equal(t, codes.InvalidArgument, grpc.Code(err))`

Should I add that check as well?